samples/unit-tests/tooltip/outside/demo.js

```diff
@@ -82,6 +82,53 @@ QUnit.test('Outside tooltip styling', function (assert) {
     );
 });
 
+QUnit.test('Tooltip with default positioner and set outside true on correct position (#16944)', assert => {
```

Comment:
I can see this bug is very much based on the margin set. What about adding one more assert with css margin or `chart.margin` being set?

Also, is it possible to reuse an already existing chart instead of creating a new `QUnit.test` with a new chart? We already have thousands of tests and if we will create a separate chart for every bug, we will soon end up with very low performance (every tests run will be very long).

Reply:
Thanks for the suggestions, I will keep in mind about test and performance.

In this case eventually, we can put in `samples/unit-tests/tooltip/position` but when I added asserts the code looked unreadable, I insert in a separate test.

Reply:
What about adding your updates and asserts to the test above?
You can rename the title from `"Outside tooltip styling"` to `"Outside tooltip styling and position"`. The chart's config is almost the same so it should be easy to combine these 2 tests into one.